backend/app/core/litellm_health.py

```
@@ -0,0 +1,303 @@
+                    # 使用 requests 库，缩短超时时间进行快速测试
+                    requests_response = requests.get(
+                        health_url,
+                        headers=headers,
+                        timeout=min(5.0, self.timeout),  # 最多5秒
+                        proxies={},  # 禁用代理
+                    )
+                    status_code = requests_response.status_code
+                    response_text = requests_response.text
+                else:
+                    # 使用 httpx 库
+                    async with httpx.AsyncClient(
```

Pylint (3.3.7)

[error] Using variable 'httpx' before assignment

(E0601)